### DISCUSSION TITLE
How to disable 'x-powered-by' header

### DISCUSSION THREAD

**Discussion by levelasquez** [vercel/next.js#11335] [Help] [answered]
Hello,

I'm having some problems trying to disable the 'x-powered-by' header.
I don't have a custom server and i'm using next 9.3.1.

next.config.js file
```
module.exports = {
  webpack: (config, { webpack }) => {
    config.plugins.push(new webpack.IgnorePlugin(/\/__tests__\//));

    return config;
  },
  poweredByHeader: false,
  env: {
    BFF_URL: process.env.BFF_URL,
    BFF_TOKEN: process.env.BFF_TOKEN,
    RECAPTCHA_KEY: process.env.RECAPTCHA_KEY,
  },
};
```

when i do `npm run dev`
![image](https://user-images.githubusercontent.com/5843706/77476330-239d0a00-6df9-11ea-88e3-03688c337834.png)

when i do `npm run build` and `npm start`
![image](https://user-images.githubusercontent.com/5843706/77476583-98704400-6df9-11ea-97e1-e0492b5781bb.png)

any solution or idea?
Thanks.

**Comment by timneutkens** [comment, marked as answer]
Maybe you're not deploying the `next.config.js` 🤔 

Overall it doesn't really matter if the header is there or not. It's super trivial to know if a site is powered by Next.js.

**Reply by levelasquez** [reply]
Oh man you right, i was not deploying the `next.config.js` now is working, thanks you a lot.

In this case our Security Team force us to disable the header.
